examples/arrays/arrays.go

```diff
@@ -31,6 +31,16 @@ func main() {
 	b := [5]int{1, 2, 3, 4, 5}
 	fmt.Println("dcl:", b)
 
+	// You can also have the compiler count the number of
+	// elements for you with `...`
+	b = [...]int{1, 2, 3, 4, 5}
+	fmt.Println("dcl:", b)
+
+	// If you specify the index with `:`, the elements in
+	// between will be zeroed.
```

Comment:
I don't think that information would add much value since gobyexample is meant for people who already have experience in other programming languages and the length of a list is always the maximum index plus one everywhere.

Reply:
Right, I mean it specifically in the context of the `[...]` syntax with explicitly specifying indeces. For example in your second `b` with the `3:`, the array length is 5 because of that.

I don't feel strongly about this, though.